css/util/color/index.css

```diff
@@ -0,0 +1,41 @@
+
+.bg-primary {
+    background-color: var(--primary-color)  !important;
```

Comment:
as vikram mentioned don't use !important unless it is very much required

Reply:
Okay!
I will make changes to that.

Reply:
without !important breaks the layout on card component.

Reply:
Could you approve this

Reply:
Fixed the breaking of layout without using !important